packages/btcindexer/db/migrations/0001_initial_schema.sql

```diff
@@ -1,17 +1,20 @@
 -- This table tracks the blocks received from the relayer (queue for cron job)
-CREATE TABLE btc_blocks (
-	height INTEGER PRIMARY KEY,
-	hash TEXT NOT NULL UNIQUE,
-	processed_at INTEGER NOT NULL, -- timestamp_ms
-	status TEXT NOT NULL DEFAULT 'new' CHECK (status IN ('new', 'scanned')) -- 'new' | 'scanned'
+CREATE TABLE IF NOT EXISTS btc_blocks (
+  hash TEXT NOT NULL,
+  height INTEGER NOT NULL,
+  network TEXT NOT NULL,
+  status TEXT NOT NULL DEFAULT 'new' CHECK (status IN ('new', 'scanned')),
```

Comment:
let's change it to boolean (true = processed). You can add a follow-up task for this week to not block this PR